src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -1820,9 +1918,9 @@ private static void validateModuleWiseTesting(Iterator<Node> itrChecks,
                 }
 
                 assertWithMessage(styleName + "_style.xml rule '" + ruleName + "' module '"
-                                + moduleName + "' has mismatched config/test links")
-                    .that(config.getTextContent().trim())
-                    .isEqualTo(configName);
+                        + moduleName + "' has mismatched config/test links")
+                        .that(config.getTextContent().trim())
+                        .isEqualTo(configName);
```

Comment:
Undo all unnecessary changes in this file.

Reply:
please do here and in all other places.

Reply:
done.